### DISCUSSION TITLE
How to add the cute octocat on the right corner?

### DISCUSSION THREAD

**Discussion by Becheler** [jothepro/doxygen-awesome-css#80] [Q&A]
Amazing repo, thank you! And the doc is 💯 
One tiny question: I can't find how to activat the cute octocat link on the top right corner with "view source on github" ?

**Comment by jothepro** [comment]
Thank you, I appreciate that you like the documentation! 😊 

It's not part of the theme. You'll have to add it on your own by modifying the doxygen header template.

I used the [GitHub Corner by Tim Holman](https://tholman.com/github-corners/) and added it [here in the `header.html`](https://github.com/jothepro/doxygen-awesome-css/blob/main/doxygen-custom/header.html#L46) with [some custom CSS](https://github.com/jothepro/doxygen-awesome-css/blob/main/doxygen-custom/custom.css#L1).
